protocol/communities/manager.go

```diff
@@ -376,6 +388,11 @@ func NewManager(identity *ecdsa.PrivateKey, installationID string, db *sql.DB, e
 		}
 	}
 
+	if managerConfig.allowForcingCommunityMembersReevaluation {
+		manager.logger.Warn("allowing forcing community members reevaluation, this should only be used in test environment")
+		manager.forceMembersReevaluation = make(map[string]chan struct{}, 10)
```

Comment:
Why 10 here? :thinking:

Reply:
This is quite random really. Should be enough for any tests.